airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_32_8_001__AirbyteConfigDatabaseDenormalization.java

```diff
@@ -0,0 +1,883 @@
+    final Field<JSONB> schedule = DSL.field("schedule", SQLDataType.JSONB.nullable(true));
+    final Field<Boolean> manual = DSL.field("manual", SQLDataType.BOOLEAN.nullable(false));
+    final Field<JSONB> resourceRequirements = DSL.field("resource_requirements", SQLDataType.JSONB.nullable(true));
+    final Field<OffsetDateTime> createdAt = DSL.field("created_at", SQLDataType.TIMESTAMPWITHTIMEZONE.nullable(false));
+    final Field<OffsetDateTime> updatedAt = DSL.field("updated_at", SQLDataType.TIMESTAMPWITHTIMEZONE.nullable(false));
+
+    ctx.createTableIfNotExists("connection")
+        .columns(id,
+            namespaceDefinition,
+            namespaceFormat,
+            prefix,
+            sourceId,
+            destinationId,
+            name,
+            catalog,
+            status,
+            schedule,
+            manual,
+            resourceRequirements,
+            createdAt,
+            updatedAt)
+        .constraints(primaryKey(id),
+            foreignKey(sourceId).references("actor", "id"),
+            foreignKey(destinationId).references("actor", "id"))
+        .execute();
+    ctx.createIndex("connection_source_id_idx").on("connection", "source_id").execute();
+    ctx.createIndex("connection_destination_id_idx").on("connection", "destination_id").execute();
+
+    LOGGER.info("connection table created");
+
+    final List<ConfigWithMetadata<StandardSync>> configsWithMetadata = listConfigsWithMetadata(
+        ConfigSchema.STANDARD_SYNC,
+        StandardSync.class,
+        ctx);
+    boolean connectionOperationCreated = false;
+    for (final ConfigWithMetadata<StandardSync> configWithMetadata : configsWithMetadata) {
+      final StandardSync standardSync = configWithMetadata.getConfig();
+      ctx.insertInto(DSL.table("connection"))
+          .set(id, standardSync.getConnectionId())
+          .set(namespaceDefinition, fromStandardSync(standardSync.getNamespaceDefinition()))
+          .set(namespaceFormat, standardSync.getNamespaceFormat())
+          .set(prefix, standardSync.getPrefix())
+          .set(sourceId, standardSync.getSourceId())
+          .set(destinationId, standardSync.getDestinationId())
+          .set(name, standardSync.getName())
+          .set(catalog, JSONB.valueOf(Jsons.serialize(standardSync.getCatalog())))
+          .set(status, StatusType.fromStandardSync(standardSync.getStatus()))
+          .set(schedule, JSONB.valueOf(Jsons.serialize(standardSync.getSchedule())))
+          .set(manual, standardSync.getManual())
+          .set(resourceRequirements, JSONB.valueOf(Jsons.serialize(standardSync.getResourceRequirements())))
+          .set(createdAt, OffsetDateTime.ofInstant(configWithMetadata.getCreatedAt(), ZoneOffset.UTC))
+          .set(updatedAt, OffsetDateTime.ofInstant(configWithMetadata.getUpdatedAt(), ZoneOffset.UTC))
+          .execute();
+      createAndPopulateConnectionOperation(ctx, !connectionOperationCreated, configWithMetadata);
```

Comment:
ah here's the problem. in the schema_dump i noticed this table doesn't exist. it is because it is never created because the creation of the table is happening in here. those two concerns need to be separated. the table should be created, even if there are no operations in the initial migration.